src/app/effects/board.effect.ts

```diff
@@ -0,0 +1,71 @@
+import { Injectable } from '@angular/core';
+import { Actions, Effect } from  '@ngrx/effects';
+import { Store } from '@ngrx/store';
+import { Notification, Notifications, NotificationType } from 'ngx-base';
+import { Observable } from 'rxjs';
+import { BoardMapper } from '../models/board.model';
+import { normalizeArray } from '../models/common.model';
+import { spaceSelector } from '../models/space';
+import { BoardService } from '../services/board.service';
+import { AppState } from '../states/app.state';
+import * as BoardActions from './../actions/board.actions';
+
+export type Action = BoardActions.All;
+
+@Injectable()
+export class BoardEffects {
+  constructor(
+    private actions$: Actions,
+    private boardService: BoardService,
+    private store: Store<AppState>,
+    private notifications: Notifications
+  ) {}
+
+  @Effect() getBoardApi: Observable<Action> = this.actions$
```

Comment:
We don't need this extra chunk as I said earlier. `this.boardService.getBoardApiUrl(spaceTemplateUrl)` can be jammed into `getBoards` itself.